buildSrc/src/main/kotlin/conventions/publishing.gradle.kts

```diff
@@ -168,7 +157,7 @@ tasks.register("publishArchives") {
         }
     }
     if (gitVersionMatch) {
-        dependsOn(tasks.withType<PublishToMavenRepository>())
+        dependsOn(tasks.named("publishToSonatype"))
```

Comment:
Should we add `closeAndReleaseSonatypeStagingRepository` or at least `closeSonatypeStagingRepository`?